### ISSUE TITLE
Remove Dedication and Acknowledgement

### ISSUE THREAD

**Issue by robmcclel** [chnm/anthologize#85]
These are much more trouble than they are worth.  It's a lot easier to just add in a part that is for front matter like this.  I highly recommend you eliminate these two areas -- or at least give the the user the option to remove them.

**Comment by boonebgorges** [comment]
Yeah, it would be nice to have at least a simple way to disable them.

**Comment by robmcclel** [comment]
Yes -- options are always good.
